grammars/julia.cson

```diff
@@ -194,6 +194,10 @@ repository:
         name: "support.function.macro.julia"
       }
       {
+        match: "[[:alpha:]_][[:word:]!]*"
+        name: "support.identifier.julia"
```

Comment:
This fails to match `B` in  `B'`.

I tried adjusting this to be:

```
      {
        match: "([[:alpha:]_][[:word:]!]*)'?"
        captures:
            "1":
                name: "support.identifier.julia"
      }
```

But it didn't help. Any ideas?